packages/salesforcedx-utils-vscode/src/commands/commandletExecutors.ts

```diff
@@ -135,11 +134,13 @@ export abstract class LibraryCommandletExecutor<T>
   constructor(
     executionName: string,
     logName: string,
-    outputChannel: vscode.OutputChannel
+    outputChannel: vscode.OutputChannel,
+    cancellable: boolean = false
```

Comment:
You can add `protected cancellable = true`; in the subclass instead of introducing this change on the base class .

Reply:
TypeScript helps but still `super(..., true)` isn't as readable.

Reply:
Great suggestion, thanks! I agree the `super(..., true)` could be a bit hard to read